gitops/openshift/apps/scss-jag/template/scss_jag.yaml

```diff
@@ -114,11 +114,11 @@ objects:
                   key: ORDS_HOST
```

Comment:
The values seem very high well probably want to reassess later. but this make the pod useable for now. The before values were bottlenecking our pod hard